testdata/formula_contrasts/SRP254919.contrasts_string.yaml

```diff
@@ -0,0 +1,13 @@
+contrasts:
+  - id: treatment_mCherry_hND6_
+    formula: "~ treatment"
+    comparison: ["treatment", "mCherry", "hND6"]
+  - id: treatment_mCherry_hND6_sample_number
+    formula: "~ treatment + sample_number"
+    comparison: ["treatment", "mCherry", "hND6"]
+    blocking_factors: ["sample_number"]
+  - id: treatment_mCherry_hND6_sample_number_complex
+    formula: "~ 0 + treatment + sample_number"
+    comparison: ["treatment", "mCherry", "hND6"]
+    blocking_factors: ["sample_number"]
```

Comment:
Are those two still needed when specifying formula and contrast?

Reply:
ok, fair enough! Could you please keep track in an issue that we still need to clean up the parameter set?